superset-frontend/src/components/InfoTooltip/index.tsx

```diff
@@ -52,6 +52,14 @@ const StyledTooltip = styled(Tooltip)`
   }
 `;
 
+const StyledTooltipTitle = styled.span`
+  display: -webkit-box;
```

Comment:
Why do we use webkit box instead of flex?

Reply:
Nevermind, I've just read https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp 👍